Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1
+
+FROM node:alpine as build
+
+ARG OLLAMA_API_BASE_URL='/ollama/api'
+RUN echo $OLLAMA_API_BASE_URL
+
+ENV PUBLIC_API_BASE_URL $OLLAMA_API_BASE_URL
+RUN echo $PUBLIC_API_BASE_URL
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./ 
+RUN npm ci
+
+COPY . .
+RUN npm run build
+
+CMD [ "npm", "run", "dev"]
```

Comment:
Thanks for the PR! we might want to use something like: https://www.npmjs.com/package/http-server instead of serving the webui using `npm run dev`!